releasenotes/notes/remove-deprecated-visualization-pulse-8adf40ff1a69df63.yaml

```
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+    Use the new interface `~qiskit.visualization.pulse_drawer` for pulse visualization.
+deprecations:
+  - |
+    The classes implemented in `~qiskit.visualization.pulse` are deprecated since 0.23.
+    The `pulse` folder is removed.
+
```

Please make your release note more verbose so that it will be easier to users to migrate. 

```suggestion
---
upgrade:
  - |
    The :class `qiskit.visualization.pulse.SchedStyle`,
    `qiskit.visualization.pulse.PulseStyle`, 
    `qiskit.visualization.pulse.matplotlib.EventsOutputChannels` 
    `qiskit.visualization.pulse.matplotlib.WaveformDrawer`,
    `qiskit.visualization.pulse.matplotlib.ScheduleDrawer`
    and :func `qiskit.visualization.pulse.interpolation.interp1d`, 
    `qiskit.visualization.interpolation.pulse.step_wise`
    are deprecated. Use the new interface `qiskit.visualization.pulse_drawer` for pulse 
    visualization.
```